tests/Migrations/20200602_create_user_counts_view.php

```diff
@@ -1,4 +1,4 @@
 <?php
 
 return Exo\ViewMigration::create('user_counts')
-    ->withBody('select count(distinct id) as user_count from test.users');
+    ->withBody('SELECT COUNT(id) AS user_count FROM users');
```

Comment:
Did you mean to drop Distinct? 
It was not dropped in tests/Migrations/20200605_create_user_counts_view_with_context.php
Unless its needed for specific testing, we should keep these consistent.